platform/darwin/src/MGLMapSnapshotter.mm

```diff
@@ -96,51 +110,61 @@ - (void)startWithCompletionHandler:(MGLMapSnapshotCompletionHandler)completion;
 - (void)startWithQueue:(dispatch_queue_t)queue completionHandler:(MGLMapSnapshotCompletionHandler)completion;
 {
     if ([self isLoading]) {
-        NSDictionary *userInfo = @{NSLocalizedDescriptionKey: @"Already started this snapshotter"};
-        NSError *error = [NSError errorWithDomain:MGLErrorDomain code:1 userInfo:userInfo];
-        dispatch_async(queue, ^{
-           completion(nil, error); 
-        });
-        return;
+        [NSException raise:@"MGLAlreadyStartedSnapshotterException"
```

Comment:
Per #7258, use one of Foundation's predefined exception names or pull this string out into a public constant.